Modules/HR/Config/config.php

```diff
@@ -41,7 +41,17 @@
     'opportunities-status' => [
         'draft' => 'Draft',
         'published' => 'Published',
-        'closed' => 'Closed'
+        'closed' => 'Closed',
+        'archived' => 'Archived',
+        'pending review' => 'Pending Review',
+    ],
+
+    'opportunities-status-mapping' => [
```

Comment:
let's rename the key
```suggestion
    'opportunities-status-wp-mapping' => [
```